### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by yamila-maker** [FavyTeam/FakeTransaction#3]
Please I need a fake bitcoin to be sent to a wallet address of $9000

**Comment by FavyTeam** [comment]
yes, we are selling private desktop app, if you are interest, just let me know.

**Comment by jatta101** [comment]
> yes, we are selling private desktop app, if you are interest, just let me know.

can you send meyour telegram or whatsaapp this my email annanjie410@gmail.com
